WebRTCPlugin/VideoRenderer.cpp

```diff
@@ -89,15 +90,15 @@ HRESULT VideoRenderer::OnDrawAdvanced(ATL_DRAWINFO& di)
 	RECT* rc = (RECT*)di.prcBounds;
 	HDC hdc = di.hdcDraw;
 
-	// Prevent any initial resizing flicker

```

Comment:
are you sure this won't cause a "flicker"?

Reply:
?

Reply:
We have initial "resizing flicker" anyway even in chrome